src/Compute/Compute/Extension/SetAzureVMExtensionBaseCmdlet.cs

```diff
@@ -40,7 +40,7 @@ public class SetAzureVMExtensionBaseCmdlet : VirtualMachineExtensionBaseCmdlet
 
         [Alias("ExtensionName")]
         [Parameter(
-            Mandatory = false,
+            Mandatory = true,
```

Comment:
Making a parameter from optional to required will cause  breaking change. Can you explain the reason?

Reply:
stayed as optional

Reply:
Looks it's still required.

Reply:
my mistake.  The Name parameter, even though it was marked optional, was in fact mandatory. Users setting the extensions without extension name would have got an error from the API.

Reply:
Please suppress exceptions from static analysis in https://github.com/Azure/azure-powershell/blob/main/tools/StaticAnalysis/Exceptions/Az.Compute/BreakingChangeIssues.csv